lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -93,6 +94,21 @@ class RenderForm extends React.Component {
     $('.form-group').removeClass('has-error');
   }
 
+  validatePhoneNumber(number, errors)
+  {
+    let is_valid_phone_number = true
+    let pattern = new RegExp("^03[0-9]{2}-[0-9]{7}$")
```

Comment:
I believe that users could possibly provide the landline number and since out support team would be contacting the users manually so we should not add restriction on the format. 

If necessary that the least we can do is to add a minimum and maximum length for this field considering the phone format `03334827335`, `0333-4827335`, `+923334827335`, `+924237498533` etc.